jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I personally think the 2nd section of this section needs a rework, because these three question are NOT the core of perturbation modeling - they just happen to be things that you can do with single cell perturbation data. Fundamentally, they are tangential to actual drug discovery and development.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I also think that these three bullet points should 100% be accompanied by something like "using [method] in [authoryear] PBMC dataset"</p>